client/src/modules/reports/generate/account_report_multiple/account_report_multiple.html

```diff
@@ -0,0 +1,62 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.REPORT_ACCOUNTS_MULTIPLE.TITLE</h3>
+      <p class="text-info" translate>REPORT.REPORT_ACCOUNTS_MULTIPLE.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+
+            <bh-account-select-multiple
+              id="account-id"
+              account-ids="ReportConfigCtrl.accountIds"
+              label="FORM.SELECT.ACCOUNTS"
+              name="account"
+              on-select-callback="ReportConfigCtrl.selectAccount(account)"
+              on-change="ReportConfigCtrl.onChangeAccounts(id)"
+              exclude-title-accounts="true"
+              required="true"
+              validation-trigger="ConfigForm.$submitted">
```

Comment:
It looks like this is the only place `bh-account-select-multiple` is used.  Could you implement https://github.com/IMA-WorldHealth/bhima-2.X/issues/3134 while you are here?

Basically, you need to remove `validation-trigger` from here, the component `bindings` and the component template itself.  You can also remove the `name` binding.

When you have done both of those, change `ng-form="{{ $ctrl.name }}"` in the [first line of the bhAccountSelectMultiple template](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/templates/bhAccountSelectMultiple.tmpl.html#L1) to `ng-form="AccountSelectMultiple"`.

Then you can change everywhere in the template that said `$ctrl.validationTrigger` and replace it with `AccountSelectMultiple.$submitted`.